config/routes.rb

```diff
@@ -3,13 +3,16 @@
   namespace :api do
     namespace :v1 do
       get '/garden_healths/search', to: 'search#index'
-      namespace :plants do 
+      namespace :plants do
         get '/search', to: 'search#index'
       end
       resources :plants
       resources :garden_healths
       resources :users
-      namespace :gardens do 
+      resources :gardens do
+        get :sensors, to: "gardens/sensors#index"
```

Comment:
I have never seen a route constructed like this before! Looks official, good work